test/epiviewpoint_web/controllers/import_controller_test.exs

```diff
@@ -11,7 +11,9 @@ defmodule EpiViewpointWeb.ImportControllerTest do
   @admin Test.Fixtures.admin()
 
   describe "create" do
-    test "prevents non-admins from uploading", %{conn: conn, user: user} do
+    @describetag :tmp_dir
```

Comment:
I've not seen [@describetag](https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-module-and-describe-tags) before, very cool!